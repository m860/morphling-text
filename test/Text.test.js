import * as React from "react";
import renderer from 'react-test-renderer';
import createText from "../src"

const Text = createText(React);

test(`render <Text/>`, () => {
    const component = renderer.create(
        <Text width={100} height={100} scale={1}>hello</Text>
    );
    expect(component.toJSON()).toMatchSnapshot();
});

test(`render <Text/> with fontSize less than 12`, () => {
    const component = renderer.create(
        <Text width={100} height={100} scale={0.5} style={{fontSize: 12}}>hello</Text>
    );
    expect(component.toJSON()).toMatchSnapshot();
});