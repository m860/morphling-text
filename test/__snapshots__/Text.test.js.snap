// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <Text/> 1`] = `
<div
  style={
    Object {
      "height": 100,
      "width": 100,
    }
  }
>
  hello
</div>
`;

exports[`render <Text/> with fontSize less than 12 1`] = `
<div
  style={
    Object {
      "fontSize": 6,
      "height": 100,
      "transform": "scale(0.5)",
      "width": 100,
    }
  }
>
  hello
</div>
`;
